src/view/popup/usePopup.ts

```diff
@@ -78,18 +78,25 @@ export const usePopup = () => {
             if (e instanceof Error) {
                 setError(e.message);
             }
-            return;
+            return false;
         }
         try {
             await calculationsRepository.setOngoingCalculation(true);
         } catch (e: unknown) {
             if (e instanceof Error) {
                 setError(e.message);
             }
-            return;
+            return false;
+        }
+        try {
+            await refreshActiveTabAndRecordBytes(userType === "new user");
+        } catch (e: unknown) {
+            setError((e as Error).message);
```

Comment:
you type it as unknown then you type cast it again - any reason for this ?

Reply:
[sadness](https://github.com/microsoft/TypeScript/issues/42596)